tasks/th/thaipop.py

```diff
@@ -0,0 +1,122 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+'''
+Bigmetadata tasks
+tasks to download and create metadata
+'''
+
+from tasks.tags import SubsectionTags, SectionTags
+from tasks.meta import (GEOM_REF, current_session, GEOM_NAME, OBSColumn)
+from tasks.util import ColumnsTask, TableTask, Carto2TempTableTask
+from collections import OrderedDict
+
+class ImportThai(Carto2TempTableTask):
+
+    subdomain = 'solutions'
+    table = 'thai_districts'
+
+
+# In[ ]:
+
+class ThaiColumns(ColumnsTask):
+
+    def requires(self):
+        return {
+            'sections': SectionTags(),
+            'subsections': SubsectionTags(),
+        }
+
+    def version(self):
+        return 1
+
+    def columns(self):
```

Comment:
Is there a column in the original table that looks like a consistent geom identifier (akin to FIPS code)?